src/DynamoCore/Migration/Migration.cs

```diff
@@ -321,22 +321,16 @@ orderby result.From
             var migrationData = new NodeMigrationData(elNode.OwnerDocument);
             migrationData.AppendNode(elNode as XmlElement);
 
-            while (workspaceVersion != null && workspaceVersion < currentVersion)
+            foreach(var migration in migrations.Where(m=>m.From >= workspaceVersion))
```

Comment:
though - `From` seems kind of strange in context when said out loud, this WorkspaceMigration migrates workspaces with versions from 2.0... I don't know how to finish this sentence to make this correct. 

Perhaps WorkspaceAttribute needs a different property name like `migrateWorkspaceVersionsBelow` ... I am not sure.

Also seems `CurrentWorkspaceVersion` is no longer used in this method.